db/citations.yml

```diff
@@ -2082,6 +2082,49 @@ boolc;pm05:
   wwwemail: ' sb2896@columbia.edu'
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/boolc_pm05.pdf
   year: '2005'
+boota;cm20:
+  ackno: This work was supported as part of the Fluid Interface Reactions, Structures
+    and Transport (FIRST) Center, an Energy Frontier Research Center funded by the
+    U.S. Department of Energy, Office of Science, and Office of Basic Energy Sciences.
+    C. C. was supported by the Chinese Scholarship Council. L.Y. and S.J.L.B.’s effort
```

Comment:
is this the most recent version?  It seems to have my initials in it.

Reply:
Sorry I don't quite know.. He never shared the recent version with me... I will email him right now and let him double check with this.....

Reply:
ok.  If it can be corrected, good, but don't make a big deal about it if the paper is already finalized.  I doubt anyone who cares will see it.

Reply:
I have contacted Boota, he sent me the most updated acknow, and he will change " L.Y. and S.J.L.B.’s effort" into "L.Y.'s effort" once he receive the proof edits from journal.

I pushed the updates.